anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/mapper/es/ExpressionTree.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.anomdetect.mapper.es;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ExpressionTree {
```

Comment:
can you name it as Expression instead of ExpressionTree? I think I made this naming mistake in detector-mapper. name Expression seems more appropriate here.